src/components/inputs/date/date.component.tsx

```diff
@@ -29,8 +30,9 @@ const DateField: React.FC<FormFieldProps> = ({ question, onChange, handler, prev
     return false;
   }, [encounterContext.sessionMode, question.readonly, question.inlineRendering, layoutType, workspaceLayout]);
 
-  const onDateChange = ([date]) => {
-    const refinedDate = date instanceof Date ? new Date(date.setHours(0, 0, 0, 0)) : new Date(date);
+  const onDateChange = (date: CalendarDate) => {
+    const refinedDate = new Date(date.toString());
+    refinedDate.setHours(0, 0, 0, 0);
```

Comment:
Do we still need this?

Reply:
I'm nervous about anything that zeroes out the time. I think this is very likely to create issues where the date is something other than what the user selected depending on what timezone they're located in, unless we're strictly making sure to treat this value as a value in the UTC timezone.

Reply:
Yeah I think @samuelmale is right, we can probably get rid of this since we have a time picker now